jwst/wfs_combine/wfs_combine.py

```diff
@@ -139,8 +139,9 @@ def create_aligned_2(self):
             # 1a. create image to smooth by first setting bad DQ pixels equal
             #     to mean of good pixels
             data_1 = self.input_1.data.astype(np.float64)
-            wh_bad_1 = np.where(self.input_1.dq != 0)
-            wh_good_1 = np.where(self.input_1.dq == 0)
+            wh_bad_1 = np.where(np.bitwise_and(self.input_1.dq, dqflags.pixel['DO_NOT_USE']) == 1)
+            wh_good_1 = np.where(np.bitwise_and(self.input_1.dq, dqflags.pixel['DO_NOT_USE']) == 0)
+
```

Comment:
Need similar change as above, but in this case you just want:

wh_good_1 = np.where(np.logical_not(np.bitwise_and(self.input_1.dq, dqflags.pixel['DO_NOT_USE'])))